approval_frame/approval_frame_cron

```diff
@@ -0,0 +1 @@
+0 0 * * * python mailchimp_api.py
```

Comment:
Is this working for you ? We need to run this script in the django context. I think following django-extensions runscript for this might be good. http://django-extensions.readthedocs.io/en/latest/runscript.html . The structure of having a scripts directory in the top level would also be good instead of using a mail method in the api module. Let me know what you  think.